### PR TITLE
Better README file

### PR DESCRIPTION
**Pull request by taubyte0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,87 @@
...
+```
+
 # Usage
-Check the `examples` folder.
+First start by creating an instance of seer
+```go
+s := New(SystemFS("config/"))
+```
+
+Note tha you can also use a virtual file system
```
Comment (arontaubyte) [inline README.md:27]: Note that

File: README.md [hunk 2]
```diff
@@ -1,14 +1,87 @@
...
+## Features
  - Preserves comments and formatting of original document
  - Creates non existing documents
  - Maps folders and files to objects
 
+
+Note: Under the hood we use *gopkg.in/yaml.v3* so YAML 1.1 & 1.2 are supported.
+
+# Installation
+The import path for the package is *gothub.com/taubyte/go-seer*.
```
Comment (tafseer-khan) [inline README.md]: gothub?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,87 @@
...
+
+To install it, run:
+```bash
+go get gothub.com/taubyte/go-seer
```
Comment (tafseer-khan) [inline README.md]: gothub?!?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,87 @@
 # taubyte/go-seer
 
-Seer is a powerfull tool to parse and edit Yaml files.
+Seer is a powerfull tool to parse and edit Yaml files in a folder as if they represent one structure.
```
Comment (tafseer-khan) [inline README.md:3]: Go-seer is a tool to parse and edit YAML files in a directory as one structure.

if you want to add powerful, powerful has one L

File: README.md [hunk 5]
```diff
@@ -1,14 +1,87 @@
...
+The import path for the package is *gothub.com/taubyte/go-seer*.
+
+To install it, run:
+```bash
+go get gothub.com/taubyte/go-seer
+```
+
 # Usage
-Check the `examples` folder.
+First start by creating an instance of seer
```
Comment (tafseer-khan) [inline README.md:22]: of go-seer

File: README.md (hunk 1 above)
Comment (tafseer-khan) [inline README.md]: Note that you can also use a virtual file system.

**Review by tafseer-khan — CHANGES_REQUESTED** [review]

**Review by tafseer-khan — APPROVED** [review]

**Pull request merged**